### ISSUE TITLE
Can't inspect `accessibilityLabel()`

### ISSUE THREAD

**Issue by nrivard** [nalexn/ViewInspector#9]
It appears that because the path for `accessibilityLabel()` uses `elements|some`, I always get a thrown error when trying to read it. In my inspection of the inspector (sorry!) I have the following modifier:

```
    ▿ modifier : AccessibilityAttachmentModifier
      ▿ attachment : Optional<AccessibilityAttachment>
        ▿ some : AccessibilityAttachment
          ▿ properties : AccessibilityProperties
            ▿ plist : [TypedValueKey = Optional(SwiftUI.AccessibilityValue(value: nil, description: Optional(SwiftUI.Text(storage: SwiftUI.Text.Storage.verbatim("123456789012"), modifiers: [])))), LabelKey = Optional(SwiftUI.Text(storage: SwiftUI.Text.Storage.verbatim("UPCA product code"), modifiers: [])), TraitsKey = AccessibilityTraitStorage(mask: SwiftUI.AccessibilityTraitStorage.(unknown context at $7fff2c6c4d64).TraitSet(rawValue: 288), values: SwiftUI.AccessibilityTraitStorage.(unknown context at $7fff2c6c4d64).TraitSet(rawValue: 288))]
              ▿ elements : Optional<Element>
                ▿ some : TypedValueKey = Optional(SwiftUI.AccessibilityValue(value: nil, description: Optional(SwiftUI.Text(storage: SwiftUI.Text.Storage.verbatim("123456789012"), modifiers: []))))
```

As you can see, `LabelKey` is in `plist`, but does not appear in `plist.elements.some`. Only my `accessibilityValue()` is in elements. Since `accessibilityValue()` is _also_ in `plist`, I think the appropriate fix is to simply chop off `elements|some` in the path for `accessibilityElement(_::::)`

**Comment by nrivard** [comment]
An alternative is to make `LabelKey` (and `TraitStorage` etc.) also appear in `elements.some`. Not sure why they don't but I haven't fully familiarized myself with the inner workings here.

**Comment by nalexn** [comment]
Hey @nrivard 

How do you attach the `accessibilityLabel` in the code you're testing? I'd help me to troubleshoot faster, as the test I have for `accessibilityLabel()` succeeds and it's hard to compare the `dump`s

**Comment by nalexn** [comment]
Yours `dump`:

```
▿ modifier : AccessibilityAttachmentModifier
      ▿ attachment : Optional<AccessibilityAttachment>
        ▿ some : AccessibilityAttachment
          ▿ properties : AccessibilityProperties
            ▿ plist : [TypedValueKey = Optional(AccessibilityValue(value: nil, description: Optional(Text(storage: Text.Storage.verbatim("123456789012"), modifiers: [])))), LabelKey = Optional(Text(storage: Text.Storage.verbatim("UPCA product code"), modifiers: [])), TraitsKey = AccessibilityTraitStorage(mask: AccessibilityTraitStorage.(unknown context at $7fff2c6c4d64).TraitSet(rawValue: 288), values: AccessibilityTraitStorage.(unknown context at $7fff2c6c4d64).TraitSet(rawValue: 288))]
              ▿ elements : Optional<Element>
                ▿ some : TypedValueKey = Optional(AccessibilityValue(value: nil, description: Optional(Text(storage: Text.Storage.verbatim("123456789012"), modifiers: []))))
```
The `dump` from the test in the framework:
```
modifier: AccessibilityAttachmentModifier
    ▿ attachment: Optional(AccessibilityAttachment.properties(AccessibilityProperties(plist: [LabelKey = Optional(Text(storage: Text.Storage.verbatim("abc"), modifiers: []))])))
      ▿ some: AccessibilityAttachment.properties
        ▿ properties: AccessibilityProperties
          ▿ plist: [LabelKey = Optional(Text(storage: Text.Storage.verbatim("abc"), modifiers: []))]
            ▿ elements: Optional(LabelKey = Optional(Text(storage: Text.Storage.verbatim("abc"), modifiers: [])))
              ▿ some: LabelKey = Optional(Text(storage: Text.Storage.verbatim("abc"), modifiers: [])) #0
                ▿ super: PropertyList.Element
                  - keyType: AccessibilityProperties.(unknown context at $7fff2c6b8688).LabelKey #1
                  - before: nil
                  - after: nil
                  - length: 1
                  ▿ keyFilter: BloomFilter
                    - value: 137439084544
                ▿ value: Optional(Text(storage: Text.Storage.verbatim("abc"), modifiers: []))
                  ▿ some: Text
                    ▿ storage: Text.Storage.verbatim
                      - verbatim: "abc"
                    - modifiers: 0 elements
```

The inner structure has a different structure, meaning that the code for attaching the modifier, or something in the project's configs or even the Phone's accessibility settings is different.

**Comment by nrivard** [comment]
Here is my particular use-case:

```swift
    public var body: some View {
        UPCBarView(digits: upc.digits)
            .accessibility(addTraits: [.isImage, .isStaticText])
            .accessibility(value: Text(verbatim: upc.value))
            .accessibility(label: Text(upc.accessibilityLabel))
    }
```

Only the last modifier shows up in the `elements` path, so `label` succeeds and a query for `value` fails. If I switch `label` and `value`, then `value` succeeds and `label` will fail. Here is my test:

```swift
import XCTest
import ViewInspector
@testable import UPCPrinter

final class AccessiblityTests: XCTestCase {

    func testAccessilibity() throws {
        let value = "123456789012"
        let upc = try XCTUnwrap(UPCFactory.upc(from: value, verifyChecksum: false))

        let upcViewInspection = try UPCView(upc: upc).inspect().view(UPCBarView.self)
        XCTAssertEqual(try upcViewInspection.accessibilityLabel().string(), upc.accessibilityLabel)
        XCTAssertEqual(try upcViewInspection.accessibilityValue().string(), upc.value)
    }
}
```

**Comment by nalexn** [comment]
Got it! Accessibility modifiers work in a bit different way in SwiftUI than the others. Usually, the modifiers form a stack of wrapped `ModifiedContent` structures, while Accessibility patch one `ModifiedContent` and even change the inner structure. I'll implement a patch shortly. Thank you for rasing the issue!

**Comment by nrivard** [comment]
Awesome, thank you! This project is great by the way!

**Comment by nalexn** [comment]
Should be fixed in `v0.3.6`

**Comment by nrivard** [comment]
Indeed fixed, thank you! Feel free to close this issue.
